### ISSUE TITLE
[SuperTextField]- Feature: Ability to use SuperEditor plugins

### ISSUE THREAD

**Issue by Rimantovas** [superlistapp/super_editor#2252]
**Your Info**
Correct if I'm wrong, but currently there is no way to use plugins such as PatternTagPlugin with SuperTextField

**Feature Value**
This would provide ability for much more customisation regarding SuperTextField

**Comment by matthew-carroll** [comment]
You're correct. Super Editor plugins only apply to Super Editor. Once we've stabilized all the various APIs in Super Editor that relate to plugins and reactions, we can look at bringing that to text fields, too.
